Microsoft.Toolkit.Uwp.SampleApp/Shell.xaml.cs

```diff
@@ -155,6 +156,29 @@ public void RegisterNewCommand(string name, RoutedEventHandler action)
             while (innerTextbox == null);
         }
 
+        public async Task RefreshXamlRenderAsync()
+        {
+            if (_currentSample != null)
+            {
+                var code = string.Empty;
+                if (_currentSample.PropertyDescriptor != null &&
+                    _currentSample.PropertyDescriptor.Options.Count > 0 &&
+                    InfoAreaPivot.SelectedIndex == 0)
+                {
+                    code = _currentSample.BindedXamlCode + " ";
```

Comment:
What's the extra space for?

Reply:
Nothing, removed (I was trying something and forgot to remove)